js-basic/Array/array.js

```diff
@@ -0,0 +1,55 @@
+let a = [18, 23, 23, 4, 4, 5];
+let k = ['Aditya', 'vinod', 'rakesh', 'crab'];
```

Comment:
Variables name should be meaningful an informative check and change
ex:- let nameArr = ['Aditya', 'vinod', 'rakesh', 'crab'];

Reply:
Already done